lineax/_solver/tridiagonal.py

```diff
@@ -12,11 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from functools import partial
```

Comment:
Style nit, `import functools as ft; ft.partial` is the style I used throughout the Equinox ecosystem.